xknx/xknx.py

```diff
@@ -53,6 +53,7 @@ def __init__(
         self.state_updater = StateUpdater(self)
         self.knxip_interface = None
         self.started = asyncio.Event()
+        self.connected = asyncio.Event()
```

Comment:
I like this. We can then probably remove the one in the HA plugin (it also has a connected flag in the `KNXModule`) that works different